This patch contains all the Debian-specific
changes mixed together. To review them
separately, please inspect the VCS history
at http://git.debian.org/?=collab-maint/foo.git

--- /dev/null
+++ ros-fuerte-pcl-1.5.2/stack.xml
@@ -0,0 +1,42 @@
+<stack>
+  <name>pcl</name>
+  <version>1.5.2</version>
+  <description>The Point Cloud Library (or PCL) for point cloud processing - development
+    The PCL framework contains numerous state-of-the art algorithms including
+    filtering, feature estimation, surface reconstruction, registration, model
+    fitting and segmentation.</description>
+  <author email="rusu@willowgarage.com">Radu Rusu</author>
+  <maintainer email="mdixon@willowgarage.com">Michael Dixon</maintainer>
+  <license>BSD</license>
+  <copyright>Willow Garage</copyright>
+  <url>http://pointclouds.org/</url>
+
+  <build_depends>boost</build_depends>
+  <build_depends>catkin</build_depends>
+  <build_depends>cmake</build_depends>
+  <build_depends>common_msgs</build_depends>
+  <build_depends>eigen</build_depends>
+  <build_depends>flann</build_depends>
+  <build_depends>libmysqlclient-dev</build_depends>
+  <build_depends>libqhull</build_depends>
+  <build_depends>libusb-1.0-dev</build_depends>
+  <build_depends>libvtk</build_depends>
+  <build_depends>python-sphinx</build_depends>
+  <build_depends>roscpp_core</build_depends>
+  <build_depends>std_msgs</build_depends>
+
+  <depends>boost</depends>
+  <depends>cmake</depends>
+  <depends>common_msgs</depends>
+  <depends>eigen</depends>
+  <depends>flann</depends>
+  <depends>libmysqlclient-dev</depends>
+  <depends>libqhull</depends>
+  <depends>libusb-1.0-dev</depends>
+  <depends>libvtk</depends>
+  <depends>python-sphinx</depends>
+  <depends>roscpp_core</depends>
+  <depends>std_msgs</depends>
+
+  <build_type file="bloom/rules.em">custom</build_type>
+</stack>
--- /dev/null
+++ ros-fuerte-pcl-1.5.2/bloom.conf
@@ -0,0 +1,4 @@
+[bloom]
+	upstream = http://svn.pointclouds.org/ros/branches/pcl-1.5.x
+	upstreamtype = svn
+
--- /dev/null
+++ ros-fuerte-pcl-1.5.2/bloom/rules.em
@@ -0,0 +1,30 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+# Sample debian/rules that uses debhelper.
+# This file was originally written by Joey Hess and Craig Small.
+# As a special exception, when this file is copied by dh-make into a
+# dh-make output file, you may use that output file without restriction.
+# This special exception was added by Craig Small in version 0.37 of dh-make.
+
+# Uncomment this to turn on verbose mode.
+export DH_VERBOSE=1
+export DH_OPTIONS=-v
+export CMAKE_PREFIX_PATH="/opt/ros/fuerte"
+CMAKE_FLAGS = \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DUSE_ROS=ON
+%:
+	dh  $@@
+
+override_dh_auto_configure:
+	dh_auto_configure -Scmake -- \
+		-DCMAKE_INSTALL_PREFIX="/opt/ros/fuerte" \
+		-DCMAKE_PREFIX_PATH="/opt/ros/fuerte" \
+		$(CMAKE_FLAGS)
+
+override_dh_auto_test:
+	echo -- Running tests. Even if one of them fails the build is not canceled.
+	-cat /proc/meminfo
+	-cat /proc/cpuinfo
+	dh_auto_test || true
+
